src/core/valuemapmodelbase.cpp

```diff
@@ -33,11 +33,11 @@ QVariant ValueMapModelBase::map() const
 void ValueMapModelBase::setMap( const QVariant &map )
 {
   mMap.clear();
-  // QGIS 3
+
   const QVariantList list = map.toList();
-  if ( !list.empty() )
+  if ( !list.empty() ) // QGIS 3
```

Comment:
What `QGIS 3` means here?

Reply:
QGIS 3.0 vs QGIS 2.0

Reply:
Do we need comments that something is QGIS 3.0 API and not QGIS 2.0, since we are waiting for QGIS 4 to get released in the next year or so?